gcs/CONFIGURATION.md

```diff
@@ -245,6 +245,15 @@ is `false`.
     The client credential file for GCS access in the OAuth 2.0 installed
     application flow (when `fs.gs.auth.service.account.enable` is `false`).
 
+#### Impersonate service account
```

Comment:
Should we move it as a **Service account impersonation** subsection under **Authentication** section?

Reply:
It's already under Authentication if you look at the markdowns above. Updated name.